source/solvers/navier_stokes_base.cc

```diff
@@ -1484,6 +1490,13 @@ NavierStokesBase<dim, VectorType, DofsType>::postprocess_fd(bool firstIter)
                           << std::setprecision(
                                simulation_control->get_log_precision())
                           << error_velocity << std::endl;
+              if (this->simulation_parameters.multiphysics.cahn_hilliard)
+                {
+                  this->pcout << "L2 error pressure : "
+                              << std::setprecision(
+                                   simulation_control->get_log_precision())
+                              << error_pressure << std::endl;
+                }
```

Comment:
Why is the pressure only displayed for CH?

Reply:
I thought, since it was not implemented before that there was a specific reason for it, so I left it only for the Cahn-Hilliard simulations

Reply:
The issue is that for the VOF solver the pressure is sometimes erratic (it will translate by a constant) so generally pressure in VOF is not a very portable variable. I'M not sure if it's a good idea to enable it in a test. We can try and if it is portable we will keep it, if no, we can remove it in the future.